docs/user-documentation/moderne-platform/how-to-guides/reporting.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Instructions for downloading reports
+---
+
+# Reporting
+
+## Overview
+
+Moderne offers some reports:
+
+* **Usage by organization** - every recipe run and associated commit by every user for a specific organization. The generated report combines the commit and recipe run reports. This report includes
+the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run. If any commit was created in association with the recipe run details
+of the commit are included: the type of commit, status of the commit job, as well as other statistics of the commit job.
```

Comment:
You immediately go from this section into the "usage by organization" report section, which has less detail in it than these lines. I would rather more detail in the "usage by organization" report section.


Maybe something like this:


```suggestion
Moderne offers the following report:

* **Usage by organization** - every recipe run, associated commit, and time saved by every user for a specific organization. 
```

Then put this detail:

> The generated report combines the commit and recipe run reports. This report includes
the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run. If any commit was created in association with the recipe run details
of the commit are included: the type of commit, status of the commit job, as well as other statistics of the commit job.

below in the report detail section.

Reply:
I move the details further down, I also did some rewording.